tests/parser_tests.py

```diff
@@ -527,6 +530,25 @@ def test_parse_with_extra_words_at_start_and_end_valid(self):
             datetime(2016, 5, 16, 4, 5, 6, 789120),
         )
 
+        self.assertEqual(
```

Comment:
Let's place these in a new test function and comment it to indicate that it is a regression test for issue #701.